Workbooks/Azure Active Directory/AppSigninHealth/appsigninhealth.workbook

```diff
@@ -0,0 +1,171 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# App sign-in health"
+      },
+      "customWidth": "70",
+      "name": "text - 1"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "4b6af9f3-59ee-47f4-8b42-8d70295a437f",
+            "version": "KqlParameterItem/1.0",
+            "name": "Workspace",
+            "type": 5,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "value": [
+              "value::all"
+            ],
+            "typeSettings": {
+              "resourceTypeFilter": {
+                "microsoft.operationalinsights/workspaces": true
+              },
+              "additionalResourceOptions": [
+                "value::all"
+              ]
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            }
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "customWidth": "30",
+      "name": "parameters - 5"
+    },
+    {
+      "type": 11,
+      "content": {
+        "version": "LinkItem/1.0",
+        "style": "paragraph",
+        "links": [
+          {
+            "id": "5e7375d4-8e85-4fea-bb2f-605ddf4f7bf4",
+            "cellValue": "https://aka.ms/aadalertsdocs",
+            "linkTarget": "Url",
+            "linkLabel": "Learn more.",
+            "preText": "Monitor the number of successful sign-ins to your applications each hour compared to the same time the previous week. You can set automated alerts in case the number of successful sign-ins changes significantly, or if the failure rate exceeds a certain threshold.  ",
```

Comment:
this seems like a very targetted, 1 visualization workbook. I think we need to enhance the scope of this.